colossus-tests/src/test/scala/colossus/protocols/http/HttpHeadSpec.scala

```diff
@@ -91,7 +90,7 @@ class HttpHeadSpec extends WordSpec with MustMatchers {
 
   "DateHeader" must {
 
-    val formatter        = new SimpleDateFormat(DateHeader.DATE_FORMAT)
+    val formatter        = DateHeader.createFormatter()
```

Comment:
Test is here. In fact this is what inspired the refactoring in the first place

Reply:
I think @benblack86 means it might make sense to have another test that compares against a hardcoded date, since these tests passed with the incorrect format as well.